OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -368,14 +365,14 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 
 		int ISpeedModifier.GetSpeedModifier()
 		{
-			return 100 - (100 - Info.FullyLoadedSpeed) * contents.Values.Sum() / Info.Capacity;
+			return 100 - (100 - Info.FullyLoadedSpeed) * Fullness / 100;
```

Comment:
integer division. can't tell if this will always go right?

Reply:
I had made sure the division is the last action